src/js/flows/ingestFiles.js

```diff
@@ -51,16 +51,23 @@ const validateInput = (paths) => ({
 
 const createDir = () => ({
   async do({dataDir}) {
-    await fsExtra.ensureDir(dataDir)
+    dataDir && (await fsExtra.ensureDir(dataDir))
   },
   async undo({dataDir}) {
-    await fsExtra.remove(dataDir)
+    dataDir && (await fsExtra.remove(dataDir))
```

Comment:
And if they did provide a directory where all their spaces will go, it seems like we would not want to delete it for them.